src/app/preferences/preferences.controller.js

```diff
@@ -5,15 +5,26 @@
   .module('roboconf.preferences')
   .controller('PreferencesController', preferencesController);
 
-  preferencesController.$inject = ['rClient', 'rPrefs', '$scope', '$timeout', '$translate'];
-  function preferencesController(rClient, rPrefs, $scope, $timeout, $translate) {
+  preferencesController.$inject = ['rClient', 'rPrefs', '$scope', '$timeout', '$translate', '$window'];
+  function preferencesController(rClient, rPrefs, $scope, $timeout, $translate, $window) {
```

Comment:
I add $window to come back to preferences page. I think it's not necessary. I'll remove it.